libs/api-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ApiBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('keboola_api');
+
+        $treeBuilder->getRootNode()
+            ->children()
+                ->scalarNode('app_name')
+                    ->defaultValue('%app_name%')
+                ->end()
+                ->scalarNode('storage_api_url')
+                    ->cannotBeEmpty()
+                    ->defaultValue('%env(STORAGE_API_URL)%')
```

Comment:
nechtělo by se ti použít HOSTNAME_SUFFIX a getConnectionServiceUrl https://github.com/keboola/platform-libraries/pull/94/files#diff-f093f567906493e8e6f45d43f31b587d524123987ab6ba2509b7cccfde2d7cbaR79 ? Nebo ti to t přijde z prdele?